code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,773 @@
+	<b>5:</b> Open the exterior output blast doors so the turbine does not overheat.<br>
+	<b>6:</b> Initiate the turbine boot up procedure on the console and configure the SMES units to evenly match the output.<br><br>
+	Best of luck to you! With me out - you're in charge of keeping this place patched up and running.<br><br>
+	-<i>Quartermaster Lindsey.</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/note5
+	name = "paper - 'Stained Note'"
+	info = {"<br>
+	To whom ever is next on janitorial duty:<br><br>
+	The disposal piping under dormitories appears to have clogged again, if the complaints about the odd smells are anything to go about.<br><br>
+	Flush down some heavy duty cleaner and hope it clears it up so we don't have to bother the engineering staff again.<br><br>
+	-<i>Lindsey.</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/note6
+	name = "paper - 'Torn Report'"
+	info = {"<br>
+	<b>Warden's Report, Cell 6 Incident.</b><br><br>
+	<i>Somehow the prisoner in cell 6 managed to aquire a set of tools and kitchen utensils that they then used to, over the span of several-</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/note7
+	name = "paper - 'Torn Research Report'"
+	info = {"<br><br><br><br>
+	<i>-n further testing, <b>subject 701</b> appears to somehow developed a mutation rendering it highly resistant to viral pathogen. It has been locked up with other specimen, pending transport fo-</i><br>
+	"}
+
+/obj/item/paper/dsbssigma/researchnotes
+	name = "paper - 'syndicate research notes'"
+	info = "<b>The notes appear gibberish to you. Perhaps a destructive analyzer in R&D could make sense of them.</b>"
+	origin_tech = "syndicate=3"
+
+/obj/item/card/id/away/dsbssigma
+	name = "syndicate ID card"
+	desc = "A slightly faded black and red access card. Only oddity you can make out is the word 'SIGMA' printed at the bottom right corner of the card."
+	icon = 'icons/obj/card.dmi'
+	icon_state = "syndie"
+	access = list(ACCESS_AWAY_SIGMA01)
+
+/obj/item/card/id/away/dsbssigma/private
+	registered_name = "William Simmons"
+	assignment = "Private"
+	access = list(ACCESS_AWAY_SIGMA01, ACCESS_AWAY_SIGMA02)
+
+/obj/item/card/id/away/dsbssigma/officer
+	registered_name = "Xiu Yao"
+	assignment = "Officer"
+	access = list(ACCESS_AWAY_SIGMA01, ACCESS_AWAY_SIGMA02, ACCESS_AWAY_SIGMA03)
+
+/obj/item/organ/internal/heart/gland/heals/weak
+	name = "fleshy growth"
+	desc = "A nausea-inducing growth of flesh."
+	origin_tech = "biotech=5"
+
+/obj/item/organ/internal/heart/gland/heals/weak/activate()
+	to_chat(owner, "<span class='notice'>You feel slightly revitalized.</span>")
+	owner.adjustToxLoss(-10)
+	owner.adjustBruteLoss(-10)
+	owner.adjustOxyLoss(-10)
+	owner.adjustFireLoss(-10)
+
+/obj/item/ammo_casing/c10mm/spent
+	projectile_type = null
+
+/obj/item/ammo_casing/c45/spent
+	projectile_type = null
+
+////// MACHINARY //////
+
+/obj/machinery/computer/id_upgrader/dsbssigma
+	icon_keyboard = "laptop_key"
+	access_to_give = list(ACCESS_AWAY_SIGMA01)
```

Comment:
Why not re-use ACCESS_AWAY01, and add ACCESS_AWAY02, ACCESS_AWAY03 etc for any additional accesses?
Multiple gateways cannot be loaded at once during the same round, so I don't think there's any reason for each mission to have its own unique access defines?

Reply:
Done.